Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "bootsnap", require: false
-gem "i18n"
+gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
```

Comment:
why we introduce this debug gem?

Reply:
The require: "debug/prelude" ensures it loads the necessary debugging prelude to start a debugging session. Bhai, as far as i remember, i was facing some error, adding this gem omitted those. Thank you

Reply:
If it's related to debugging shouldn't we add this in development group?

Reply:
Updated this bhai